src/commands/setup-web.js

```diff
@@ -9,10 +9,17 @@ var requirePermissions = require("../requirePermissions");
 
 var JS_TEMPLATE = fs.readFileSync(__dirname + "/../../templates/setup/web.js", "utf8");
 
+/**
+ * This command is deprecated in favor of `apps:sdkconfig web` command
+ * TODO: Remove this command after launching `apps:sdkconfig` command
+ */
 module.exports = new Command("setup:web")
-  .description("display this project's setup information for the Firebase JS SDK")
+  .description("[DEPRECATED] display this project's setup information for the Firebase JS SDK")
```

Comment:
```suggestion
  .description("[DEPRECATED: use `apps:sdkconfig web`] display this project's setup information for the Firebase JS SDK")
```

Make it easy to read the help text and figure out where to go.

Reply:
Done